src/ngMessageFormat/messageformat.js

```diff
@@ -0,0 +1,938 @@
+'use strict';
+
+var angular = window['angular'];
+
+var $$messageFormatMinErr = angular['$$minErr']('$$messageFormat');
+
+// NOTE: ADVANCED_OPTIMIZATIONS mode.
+//
+// This file is compiled with Closure compiler's ADVANCED_OPTIMIZATIONS flag! Be wary of using
+// constructs incompatible with that mode.
+
+/**
+ * @ngdoc service
+ * @name $$messageFormat
+ *
+ * @description
+ * Angular internal service to recognize MessageFormat extensions in interpolation expressions.
+ * For more information, see:
+ * https://docs.google.com/a/google.com/document/d/1pbtW2yvtmFBikfRrJd8VAsabiFkKezmYZ_PbgdjQOVU/edit
+ */
+
+var noop = angular['noop'],
+    isFunction = angular['isFunction'],
+    toJson = angular['toJson'];
+
+function stringify(value) {
+  if (value == null) { return ''; }
+  switch (typeof value) {
+    case 'string':     return value;
+    case 'number':     return '' + value;
+    default:           return toJson(value);
+  }
+}
+
+var PARSE_CACHE_FOR_TEXT_LITERALS = new Object(null);
+var PARSE_CACHE_FOR_INTERPOLATIONS = new Object(null);
+
+function parseTextLiteral(text) {
+  var cachedFn = PARSE_CACHE_FOR_TEXT_LITERALS[text];
+  if (cachedFn != null) {
+    return cachedFn;
+  }
+  function parsedFn(context) { return text; };
+  var unwatch;
+  parsedFn['$$watchDelegate'] = function watchDelegate(scope, listener, objectEquality) {
+    unwatch = scope['$watch'](noop,
+        function () {
+          if (isFunction(listener)) { listener.call(null, text, text, scope); }
+          unwatch();
+        },
+        objectEquality);
+    return unwatch;
+  };
+  PARSE_CACHE_FOR_TEXT_LITERALS[text] = parsedFn;
+  if (goog.DEBUG) {
+    // Only needed in order to pretend to be $interpolate for tests copied from interpolateSpec.js
```

Comment:
Ah.  I knew there was something I was missing but hadn't found any helpful comments to guide me when I had grepped our code.

Yes, `goog.DEBUG` is `false` *only* in the minified version and if `true` otherwise which should allow Protractor to work correctly.  I went ahead and renamed the symbol to `goog.MINIFIED` and switched the bool value to make this clear.  In the minified build, because I'm using `ADVANCED_OPTIMIZATIONS`, the entire `if` block is optimized away.